### PR TITLE
chaplain cage no longer balds you

### LINKED ISSUE #53860: Chaplain's Cage sprites
## Reproduction: get chaplain scholar kit, equip the cage.

Not bald.

Bald.

Unplayable.

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Fix chaplain cage from balding you

closes: #53860

## Why It's Good For The Game
fix consistency issue

## Changelog
:cl:
fix: chaplain head cage no longer makes you bald
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
this isnt a fix if its intended

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -76,7 +76,7 @@
 /obj/item/clothing/head/helmet/chaplain/cage
 	name = "cage"
 	desc = "A cage that restrains the will of the self, allowing one to see the profane world for what it is."
-	flags_inv = HIDEHAIR //bald
+	flags_inv = null
```
Comment (Fikou) [inline code/game/objects/items/holy_weapons.dm:79]:
```suggestion
	flags_inv = NONE
```

**Comment by ishitbyabullet** [comment]
this nerfs bald people. -rep

**Comment by LemonInTheDark** [comment]
Seems like a feature and not a bug, why was this in the issues in the first place

**Comment by LemonInTheDark** [comment]
I'm honestly kinda confused man, it looks like it was a bug turned feature. Might just merge this anyway

**Comment by necromanceranne** [comment]
It's not even true to the source material, the cage doesn't bald you in Bloodbourne.

**Pull request merged**